wallet-test/src/test/scala/org/bitcoins/wallet/util/BitcoinSWalletTest.scala

```diff
@@ -57,21 +58,44 @@ trait BitcoinSWalletTest
       .map(_ => ())
   }
 
-  def createNewWallet(): Future[UnlockedWalletApi] = {
-
-    for {
-      _ <- config.initialize()
-      wallet <- Wallet.initialize().map {
-        case InitializeWalletSuccess(wallet) => wallet
-        case err: InitializeWalletError =>
-          logger.error(s"Could not initialize wallet: $err")
-          fail(err)
-      }
-    } yield wallet
-  }
+  private val createNewWallet: Option[Config] => () => Future[
```

Comment:
scaladoc would be nice indicating the purpose of this. My understanding is it returns a function that can create a wallet with the given config